packages/flutter/test/widgets/sliver_semantics_test.dart

```diff
@@ -158,5 +278,7 @@ void main() {
       ignoreRect: true,
       ignoreTransform: true,
     ));
+
+    semantics.dispose();
   });
 }
```

Comment:
and let's update the tool to support dumping this tree in both modes as well.

Reply:
All done. PTAL.